src/worldmap.ts

```diff
@@ -433,7 +437,27 @@ export default class WorldMap {
     if (this.ctrl.settings.formatOmitEmptyValue && value === 'n/a') {
       return `${locationName}`.trim();
     } else {
-      return `${locationName}: ${value} ${unit || ''}`.trim();
+      let fieldPrefix = '__field_';
+
+      let specialFields = [
+        fieldPrefix + this.ctrl.settings.esLocationName,
+        fieldPrefix + this.ctrl.settings.esMetric,
+        fieldPrefix + this.ctrl.settings.esGeoPoint,
+      ];
```

Comment:
This looks a bit specific to Elasticsearch, right? I see that this is your current focus and I will not block merging this PR. However, it would be nice if we can provide this feature for other data sources like InfluxDB.

Do you believe there is a chance this might break anything when not running with ES?

Reply:
The test suite passes for me. If that doesn’t seem like enough confidence I could add a guard. Do you have a dev setup for integration testing other data sources? I’d be happy to check but postgres and Es are the only tools I’ve ever done mapping from.

Reply:
> The test suite passes for me. If that doesn’t seem like enough confidence I could add a guard.

I believe it's fine. Thanks.

> Do you have a dev setup for integration testing other data sources?

Currently not. I just wanted to take a note on that for coming back to this later.